test/consuming-services.test.js

```diff
@@ -32,7 +32,7 @@ describe('Consuming Services locally', () => {
             })
         })
     }).where(`name like`, 'E%')
-    if (cds.version >= '5.9.0') {
+    if (require('semver').gte(cds.version, '5.9.0')) {
```

Comment:
Please add `semver` to the devDpendencies of this project.
(it only works at the moment due to our dev setup, where `semver` is already available).

Reply:
done

Reply:
Why not use latest `semver` 7 ?
I think you need to update the package-lock as well with `npm update`

Reply:
Because it is the latest supported by node 12. I took 7 initially.

Reply:
I have bumped semver to 7 and updated package-lock